src/components/page-configuration/index.tsx

```diff
@@ -55,19 +55,31 @@ export interface PageConfigurationProps
   appName: string;
 }
 
+function getConfigBranch(configBranch: string) {
+  return configBranch?.length > 0 ? configBranch : 'master';
+}
+
+function getRadixConfigFullName(radixConfigFullName: string) {
+  return radixConfigFullName?.length > 0
+    ? radixConfigFullName
+    : 'radixconfig.yaml';
+}
```

Comment:
remember to add return typing on methods, can also be simplified
```suggestion
function getRadixConfigFullName(radixConfigFullName: string): string {
  return radixConfigFullName || 'radixconfig.yaml';
}
```